testsuite/MDAnalysisTests/coordinates/test_pdb.py

```diff
@@ -961,3 +961,11 @@ def test_partially_missing_cryst():
     assert len(u.atoms) == 3
     assert len(u.trajectory) == 2
     assert_array_almost_equal(u.dimensions, 0.0)
+
+def test_cryst_em_warning():
+    #issue 2599  
+    with pytest.warns(UserWarning) as record:
+        u = mda.Universe(PDB_CRYOEM_BOX)
+        cur_sele = u.select_atoms('around 0.1 (resid 4 and name CA and segid A)') 
+    assert record[0].message.args[0] == "1 A^3 CRYST1 record, possibly an EM structure file"
+    assert len(cur_sele) == 0
```

Comment:
As mentioned above, because this test specifically tests for a warning, I would suggest putting this in a separate test rather than trying to test multiple things at the same time (it'll make for a cleaner error message if the test ever fails).